pkg/validations/commandvalidations/dockerexec.go

```diff
@@ -0,0 +1,24 @@
+package commandvalidations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/validations"
+)
+
+func ValidateDockerExecutable(ctx context.Context, docker validations.DockerExecutable, os string) error {
```

Comment:
should this validations be in `pkg/docker`?
IMO validations fit better in packages that inform about their domain (in general)

That said, `pkg/docker` shouldn't probably depend on `pkg/validations` so maybe we need to change that too?

Reply:
moved to pkg/validations/docker.go - where all the other docker validations are for now